invenio_users_resources/services/schemas.py

```diff
@@ -67,10 +67,16 @@ class UserSchema(BaseRecordSchema, FieldPermissionsMixin):
     def is_self(self, obj):
         """Determine if identity is the current identity."""
         current_identity = self.context["identity"]
+
+        try:
+            _id = obj.id
+        except AttributeError:
+            _id = obj.get("id")
```

Comment:
Question: can we do something like `obj.get("id") or obj.id` here?

Reply:
Done! much cleaner that way thx!